doc/compiling/windows.md

```diff
@@ -2,67 +2,82 @@
 
 ## Requirements
 
-- [Visual Studio 2015 or newer](https://visualstudio.microsoft.com)
-- [CMake](https://cmake.org/download/)
-- [vcpkg](https://github.com/Microsoft/vcpkg)
-- [Git](https://git-scm.com/downloads)
-
+-   [Visual Studio 2015 or newer](https://visualstudio.microsoft.com)
+-   [CMake](https://cmake.org/download/)
+-   [vcpkg](https://github.com/Microsoft/vcpkg)
+-   [Git](https://git-scm.com/downloads)
 
 ## Compiling and installing the dependencies
 
-It is highly recommended to use vcpkg as package manager.
+It is highly recommended to use vcpkg as package manager, installed in a location without spaces in the path for compatibility with CMake.
+
+```
+"C:\Program Filesvcpkg\vcpkg.exe" # bad, may have issues with CMake
+"C:\vcpkg\vcpkg.exe" # good, no spaces means fewer problems
+```
+
+If spaces are present, you may see errors like:
+
+```
+libtool:   error: 'Files/vcpkg/buildtrees/libiconv/x64-windows-dbg/lib/libcharset.la' is not a directory
+```
 
 After you successfully built vcpkg you can easily install the required libraries:
 ```powershell
 vcpkg install zlib zstd curl[winssl] openal-soft libvorbis libogg libjpeg-turbo sqlite3 freetype luajit gmp jsoncpp gettext[tools] sdl2 --triplet x64-windows
 ```
 
-- `curl` is optional, but required to read the serverlist, `curl[winssl]` is required to use the content store.
-- `openal-soft`, `libvorbis` and `libogg` are optional, but required to use sound.
-- `luajit` is optional, it replaces the integrated Lua interpreter with a faster just-in-time interpreter.
-- `gmp` and `jsoncpp` are optional, otherwise the bundled versions will be compiled
-- `gettext` is optional, but required to use translations.
+This command takes about 10-30 minutes to complete, depending on your device.
 
-There are other optional libraries, but they are not tested if they can build and link correctly.
+-   `curl` is optional, but required to read the serverlist, `curl[winssl]` is required to use the content store.
```

Comment:
```suggestion
-   `curl` is optional, but required to read the serverlist; `curl[winssl]` is required to use the content store.
```

This should be a semicolon. If we're fixing the docs, let's make them shine.

Reply:
`curl` is strongly recommended. It's required for anything HTTP(S). The serverlist or content browser (or the built-in update checker) are just some "mainmenu" examples of this; the Lua HTTP API also needs curl to work.

(Frankly we might consider making curl mandatory in the future?)

Reply:
```suggestion
-   `curl` is highly recommended, as it's required to read the serverlist; `curl[winssl]` is required to use the content store.
```